Python/errors.c

```diff
@@ -1322,13 +1351,20 @@ PyErr_WriteUnraisable(PyObject *obj)
         }
     }
 
+    if (err_msg_str != NULL) {
+        err_msg = PyUnicode_FromFormat("Exception ignored %s", err_msg_str);
```

Comment:
Hummm, why don't allowing the caller to control the full error message? (Instead of prepending "Exception ignored")

Reply:
@serhiy-storchaka asked me to do so :-) And when I started a draft change to convert existing PyErr_WriteUnraisable() to _PyErr_WriteUnraisableMsg(), I always started error messages with "Exception ignored ".

The hook allows to control the full error message. Only proposed _PyErr_WriteUnraisableMsg() enfoce "Exception ignored " prefix. We might add a new function later if needed to let control the full error message.

Reply:
By the way, making "Exception ignored " prefix part of the API should reduce the size of binary since constant strings are shorter (remove common "Exception ignored " prefix ;-)).